CHANGELOG.md

```diff
@@ -12,6 +12,120 @@
+
+```
+- Add new build-noziti and docker-noziti targets into Makefile ([8a1c9a0…](https://github.com/edgexfoundry/edgex-go/commit/8a1c9a01a1f2d5a8ef23fa178018208c4e23c5cc))
+- Remove consul dependency ([cbe9dac…](https://github.com/edgexfoundry/edgex-go/commit/cbe9dacb0aa5af2c2c08753c4199293e4457ef93))
+```text
+
+BREAKING CHANGE: Remove consul dependency
+
+```
+<!-- not sure -->
+<!-- - Using isNull field instead of nil reading value ([#4974](https://github.com/edgexfoundry/edgex-go/issues/4974)) ([82350b2…](https://github.com/edgexfoundry/edgex-go/commit/82350b2dc17c64c6dcaf341c305da1b72442d937)) -->
+<!-- - Enable ASLR and Full RELRO while building binaries as default ([babc124…](https://github.com/edgexfoundry/edgex-go/commit/babc124801c941d32da667ad712ca5d717a91234)) -->
+- Allow store reading as null value ([faff7e1…](https://github.com/edgexfoundry/edgex-go/commit/faff7e15d0d593107ea324478509bb94221df236))
+- Add the force query param to add device metadata API ([#4929](https://github.com/edgexfoundry/edgex-go/issues/4929)) ([1033d52…](https://github.com/edgexfoundry/edgex-go/commit/1033d5290f4fd751026052b6c6be6da923b73d23))
+- Allow rules-engine service to read its token ([#4823](https://github.com/edgexfoundry/edgex-go/issues/4823)) ([5162257…](https://github.com/edgexfoundry/edgex-go/commit/5162257f8dee1b2a6e96c0426af9cd51831dbfdc))
+- Core Keeper implementation ([e8726e3…](https://github.com/edgexfoundry/edgex-go/commit/e8726e3ae68f2a04ddd2be347a70a833a7d3d060))
+```text
+
+BREAKING CHANGE: Introduced Core Keeper as a new service for configuration and registry management
+
+```
+- Implement Get All DeviceProfile BasicInfo API ([9494c74…](https://github.com/edgexfoundry/edgex-go/commit/9494c7497b3c6dc9521145e4123e3d35bbdb8d94))
+- Add new bypassValidation query param to Add/Patch Device API ([#4797](https://github.com/edgexfoundry/edgex-go/issues/4797)) ([17930b1…](https://github.com/edgexfoundry/edgex-go/commit/17930b14352cbfb7930b1ac1271c629a4bf190e4))
+- Add OpenZiti to edgex-go ([#4777](https://github.com/edgexfoundry/edgex-go/issues/4777)) ([d2d9888…](https://github.com/edgexfoundry/edgex-go/commit/d2d98883e5139249b4399f4dc0dd90a078163ffc))
+- Allow sending notifications to authenticated EdgeX endpoints ([#4763](https://github.com/edgexfoundry/edgex-go/issues/4763)) ([a48d639…](https://github.com/edgexfoundry/edgex-go/commit/a48d6392372de4eada0148d46cda0c5092cdf95e))
+- Add new StoreForwardQueueSize metric to common App services config ([#4751](https://github.com/edgexfoundry/edgex-go/issues/4751)) ([3a892ac…](https://github.com/edgexfoundry/edgex-go/commit/3a892ac0b38b8d685f9d63feb7dde70ee06d273b))
+
+- Add core-metadata device/provisionwatcher db methods for postgres ([fd73939…](https://github.com/edgexfoundry/edgex-go/commit/fd73939b4ae038ef84b6f1c31c78656397dfc769))
+```text
+
+BREAKING CHANGE: Switched default database to PostgreSQL across all services
+
+```
```

Comment:
```suggestion
- Add PostgreSQL as the new default persistence layer ([fd73939…](https://github.com/edgexfoundry/edgex-go/commit/fd73939b4ae038ef84b6f1c31c78656397dfc769))
```text

BREAKING CHANGE: Switched default database to PostgreSQL across all services

```
```